### DISCUSSION TITLE
Initial Preprocessing Pipeline - Yukon Flats Region

### DISCUSSION THREAD

**Discussion by JatinSharma222** [YaliWang2019/AK-Satellite-Imagery-Wildfire-Prediction#37] [Show and tell]
Hi Dr. Wang and Dr. Das,
I'm Jatin Sharma, a final year BCA student interested in the Alaska Wildfire Prediction project. I've put together a proof of concept preprocessing pipeline to demonstrate my understanding of satellite imagery analysis.
What I Built
I focused on the Yukon Flats region (66-67°N, 144-146°W) using Sentinel-2 data from 2022's fire season:

Pre-fire period (June 2022)
During fire (July-August 2022)
Post-fire (September 2022)

Repository: https://github.com/JatinSharma222/Alaska-Wildfire-Proof-of-concept

The pipeline handles:
Memory-optimized image loading for Colab's 12GB RAM limit
Cloud masking using SCL band
Vegetation indices (NDVI, NBR, NDMI)
Burn severity classification following USGS standards
Change detection between time periods

Key Results
The dNBR analysis successfully identified burned areas with 63.8% showing moderate burn severity. The feature correlation analysis shows NBR and NDVI as strong predictors of fire risk, which aligns with remote sensing literature.
Visual Results:

<img width="2181" height="2082" alt="before_after_comparison" src="https://github.com/user-attachments/assets/2de44d9c-caff-4901-9afa-b5efdb2c3644" />
Figure 1: Yukon Flats before (June 2022) and after (September 2022) the fire season. The true color composites show visible landscape changes, particularly in vegetation cover and burn scars.

<img width="2278" height="1183" alt="burn_severity" src="https://github.com/user-attachments/assets/07a0718d-86d2-4d64-ab36-f03d74029da1" />
Figure 2: dNBR-based burn severity classification. The analysis identified distinct severity zones, with most affected areas showing moderate burn levels. This follows USGS/FIREMON classification standards.

<img width="1780" height="879" alt="feature_importance" src="https://github.com/user-attachments/assets/c09b3837-ee0e-4bf4-aae2-8d1a8fd62aba" />
Figure 3: Top 10 features ranked by correlation with fire risk. dNBR shows the strongest relationship (0.82), followed by pre-fire spectral bands and vegetation indices. This validates the feature selection for the CNN-LSTM model.

<img width="2269" height="879" alt="fire_risk_distribution" src="https://github.com/user-attachments/assets/2675c581-159e-408c-86db-02844b72cd63" />
Figure 4: Pixel-level fire risk classification across the study area. The distribution shows 36.2% unburned, 63.8% moderate risk, and minimal high-risk pixels, matching observed fire patterns from 2022.
Questions Before Scaling Up

Should I focus on specific high-risk regions like Interior boreal forests, or aim for broader coverage?
For the model training phase, what temporal range would you recommend? I'm thinking 5-7 years (2017-2024) across 3-5 key regions.

- Is there a preference between PyTorch and TensorFlow for the CNN-LSTM implementation?
- The README mentions 30 years of weather data. Is this already compiled, or part of the GSoC scope?
- What computational resources will be available? This determines whether I can work with full 10m resolution or need          downsampling strategies.

I'm working on integrating Sentinel-1 SAR data next, and researching ERA5 weather data integration approaches.
Looking forward to your feedback.
Jatin Sharma
